chimney/src/main/scala/io/scalaland/chimney/internal/compiletime/derivation/transformer/Configurations.scala

```diff
@@ -585,7 +609,7 @@ private[compiletime] trait Configurations { this: Derivation =>
         extractPath[PathType2].matching[Subtype]
       case ChimneyType.Path.SourceMatching(init, sourceSubtype) =>
```

Comment:
We have to leave this in the DSL runtime (bincompat), so we might as well still parse it.